packages/calypso-e2e/src/lib/pages/cart-checkout-page.ts

```diff
@@ -38,9 +39,11 @@ const selectors = {
 	// Tax information
 	countryCode: `select[aria-labelledby="country-selector-label"]`,
 	postalCode: `input[id="contact-postal-code"]`,
-	submitBillingInformationButton: `button[aria-label="Continue with the entered contact details"]`,
+	submitBillingInformationButton:
+		'[data-testid="contact-form--visible"] button.checkout-button.is-status-primary',
```

Comment:
Is this change due to I18N reasons? Otherwise, the resulting selector is not as descriptive as the original form.